src/h5web/explorer/Explorer.module.css

```diff
@@ -45,3 +50,23 @@
   margin-left: -0.375rem;
   margin-right: 0.125rem;
 }
+
+.spinner {
+  position: absolute;
+  top: 0.5rem;
+  right: 0.75rem;
+  font-size: 0.75em;
+  animation: spin 1.5s ease-in-out infinite, fade 0.1s 0.1s backwards;
```

Comment:
The fade-in animation allows the spinner to not appear when a group's metadata is fetched almost instantly, for instance when using the `MockProvider`.